modules/serverless-kafka-broker-configmap.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-configmap_{context}"]
+= Configuring Kafka broker settings
+
+You can configure the number of topic partitions, replication factor, and bootstrap servers for a Kafka broker, by creating a config map and referencing this config map in the Kafka `Broker` object.
+
+:FeatureName: Kafka broker
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource (CR) are installed on your {product-title} cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Comment:
```suggestion
* You created a project or have access to a project that has the appropriate roles and permissions to create applications and other workloads in {product-title}.
```

Reply:
ISG is very much for using "that" with restrictive clauses, and this pattern is found throughout our docs.

Peer review is ultimately advisory, though, so it's your call.